pkg/apis/kops/componentconfig.go

```diff
@@ -433,6 +433,9 @@ type KubeAPIServerConfig struct {
 
 	// CPURequest, cpu request compute resource for api server. Defaults to "150m"
 	CPURequest string `json:"cpuRequest,omitempty"`
+
+	// Amount of time to retain Kubernetes events
+	EventTTL *metav1.Duration `json:"eventTTL,omitempty" flag:"event-ttl"`
```

Comment:
Can we set the data type to match what is in the k8s codebase, so in this case `time.Duration`.

https://github.com/kubernetes/kubernetes/blob/550fb1bfc3c773febbb0811262fd7a8f7847bb20/cmd/kube-apiserver/app/options/options.go#L55

Reply:
`kops edit cluster` breaks when I use `time.Duration` instead of `*metav1.Duration` :disappointed:
```
error parsing config: v1alpha2.Cluster.Spec: v1alpha2.ClusterSpec.KubeAPIServer: v1alpha2.KubeAPIServerConfig.EventTTL: readUint64: unexpected character: �, error found in #10 byte of ...|ventTTL":"3h"},"kub|..., bigger context ...|beAPIServer":{"disableBasicAuth":true,"eventTTL":"3h"},"kubeDNS":{"provider":"CoreDNS"},"kubelet":{|...
```

Reply:
@rdrgmnzs 
`kops edit cluster` breaks when I use `time.Duration` instead of `*metav1.Duration` :disappointed:
```
error parsing config: v1alpha2.Cluster.Spec: v1alpha2.ClusterSpec.KubeAPIServer: v1alpha2.KubeAPIServerConfig.EventTTL: readUint64: unexpected character: �, error found in #10 byte of ...|ventTTL":"3h"},"kub|..., bigger context ...|beAPIServer":{"disableBasicAuth":true,"eventTTL":"3h"},"kubeDNS":{"provider":"CoreDNS"},"kubelet":{|...
```